compiler/src/dotty/tools/dotc/core/Definitions.scala

```diff
@@ -830,6 +830,8 @@ class Definitions {
 
   @tu lazy val ReflectSelectableTypeRef: TypeRef = requiredClassRef("scala.reflect.Selectable")
 
+  @tu lazy val TypeablePackageClass: Symbol = requiredPackage("scala.reflect.Typeable$package").moduleClass
```

Comment:
Do we need to keep a `TypeablePackageClass`?

Reply:
We do not.